Sources/UIAlertControllerView.swift

```diff
@@ -24,6 +27,17 @@ class UIAlertControllerView: UIView {
         self.style = style
         header.text = title
         header.font = UIFont.boldSystemFont(ofSize: 20)
+        header.numberOfLines = 0
```

Comment:
I guess I'm wrong but I thought 0 was the default?

Reply:
UILabel has indeed a default value of `1`. https://github.com/flowkey/UIKit-cross-platform/blob/31b422b2553b9ea2eef1d8d398ae4a80aebde9c2/Sources/UILabel.swift#L24

Reply:
thanks for clarifying and sorry for the noise!